# FTP::RemoveDir

string dir_path
---
bool success
